examples/tilted_bottom_boundary_layer.jl

```diff
@@ -95,9 +95,22 @@ coriolis = ConstantCartesianCoriolis(f = 1e-4, rotation_axis = ĝ)
 # _perturbations_ away from the constant density stratification by imposing
 # a constant stratification as a `BackgroundField`,
 
-B_field = BackgroundField(constant_stratification, parameters=(; ĝ, N² = 1e-5))
+N² = 1e-5 # s⁻² # background vertical buoyancy gradient
+B_field = BackgroundField(constant_stratification, parameters=(; ĝ, N² = N²))
```

Comment:
```suggestion
B∞_field = BackgroundField(constant_stratification, parameters=(; ĝ, N² = N²))
```

I realized there's an issue in our UI right now where `BackgroundField` is not actually a field, making things a little confusing (we have to redefine `B∞` below to get the actual field). I'll open a separate issue for this...